build123d/hexagon.py

```diff
@@ -1,69 +1,74 @@
-from build123d import (
-    BuildPart, BuildSketch, BuildLine, Polyline, Circle, Plane, Locations, Mode, Location,
-    mirror, make_face, extrude, add)
-
+from build123d import *
```

Comment:
Please import the specific parts that are needed. This is a significant step backwards.